### DISCUSSION TITLE
Best practices for rule tuning?

### DISCUSSION THREAD

**Discussion by flukeymcswagger** [Security-Onion-Solutions/securityonion#5039] [Unsupported Versions] [answered]
Hi folks,

Just testing securityonion 2 as a long time user of previous iterations. I'm a bit frustrated with tuning alert output via the GUI. I have two windows hosts who generate false positive alerts via the "ET USER_AGENTS Steam HTTP Client User-Agent" signature. I want to keep this alert but ignore two specific source IPs. This used to be really easy in SGUIL. Right clicking on the alert ID and selecting 'auto categorize' would bring up a dialogue box where one could add extra conditions (source IP = xxx, sourceport = any, dst ip = any, dst port = any, etc, etc). 

I've been poking around in the docs and it's not really clear what the best way to do this is. What's the best/recommended/easiest way to add specific conditions to rules, similar to SGUIL's auto-categorization feature?

**Comment by TOoSmOotH** [comment, marked as answer]
Auto cat in this example is not the correct way to handle this. The proper way to address this is with thresholding. This way you don't generate the actual data. https://docs.securityonion.net/en/2.3/managing-alerts.html?highlight=threshold#threshold

**Reply by flukeymcswagger** [reply]
Understood, and thanks for the response. To further the discussion a bit, I would love to see a GUI element that would allow an analyst to propose threshold changes from the SOC console. Perhaps a context menu for the alert.name or rule.uuid? Another option would be a third icon next to 'acknowledge|escalate' for 'adjust thresholds'. Editing YAML isn't rocket surgery, but I'd much prefer a programmatic approach to editing important config files vs. letting humans with their fat fingers do it.
